openwisp_utils/admin_theme/static/lib/netjsongraph/css/netjsongraph.css

```
@@ -0,0 +1,382 @@
+.njg-container .switch-wrap input[type="checkbox"]:checked + label:before {
+  transform: translateX(34px);
+}
+.njg-container .njg-date {
+  position: absolute;
+  right: 20px;
+  bottom: 20px;
+  z-index: 1;
+}
+
+@font-face {
+  font-family: "iconfont";
```

Stylelint (17.11.0)

[error] Expected no quotes around "iconfont" (font-family-name-quotes)

(font-family-name-quotes)